tests/AutoReview/ProjectCodeTest.php

```diff
@@ -763,12 +818,15 @@ public function testConstantsAreInUpperCase(string $className): void
 
     private function createTokensForClass(string $className): Tokens
     {
-        $file = $className;
-        $file = preg_replace('#^PhpCsFixer\\\Tests\\\#', 'tests\\', $file);
-        $file = preg_replace('#^PhpCsFixer\\\#', 'src\\', $file);
-        $file = str_replace('\\', \DIRECTORY_SEPARATOR, $file).'.php';
+        $filename = (new \ReflectionClass($className))->getFileName();
```

Comment:
back to few days ago we were using both approaches, filename to filepath manually and with reflection class. tbh, I would like to avoid reflection usage here. It is some kind of "let's do magic under the hood" when we are almost abusing the language, for me. filesystem approach is "non-magic" - we do not need to rely on some internals to tell us where is the file, file is ours, we have strict structure on filesystem, we know where to access any file for any class.